camera_matrix:
- [386.8893737792969, 0.0, 321.6220703125]
- [0.0, 386.8893737792969, 240.8651123046875]
- [0.0, 0.0, 1.0]
dist_coeffs: [0.0, 0.0, 0.0, 0.0, 0.0]
dist_model: distortion.brown_conrady
width: 640
height: 480
depth_scale: 0.0010000000474974513